packages/client/components/BottomControlBarReady.tsx

```diff
@@ -1,21 +1,21 @@
 import styled from '@emotion/styled'
 import graphql from 'babel-plugin-relay/macro'
 import useAtmosphere from 'hooks/useAtmosphere'
-import {TransitionStatus} from 'hooks/useTransition'
+import { TransitionStatus } from 'hooks/useTransition'
```

Comment:
I think `formatOnSave` is breaking some eslint rules.
i'd suggest running eslint on save.
for example:
```
"editor.codeActionsOnSave": {
    "source.organizeImports": true,
    "source.fixAll.eslint": true
},
```
i'm leaving this in the user settings vs. workspace settings because it can be a little computationally intensive if you're doing something like a replace-all across 100 files, but if you think it'd be useful to promote to the workspace settings we can do that, too.

Also, you might want to consider adding the formatting rule:

```
"typescript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false,
"javascript.format.insertSpaceAfterOpeningAndBeforeClosingNonemptyBraces": false,
```

I'm surprsied `lint-staged` isn't picking these up. are you committing with `--no-verify` by chance?

Reply:
I'm not using --no-verify. I'm not sure why it's not getting picked up. I'll go ahead and make the changes you suggested.